### PR TITLE
About Us testing and MockMetrics completed.

### PR DESCRIPTION
**Pull request by Mannyplg**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -10,8 +10,23 @@ import LeaderBoard from "./pages/LeaderBoard";
...
+    // }, []);
+  
+    // const readMetrics = () => {
+    //   fetch("/metrics")
+    //     .then((response) => response.json())
+    //     .then((payload) => {
+    //       setApartments(payload);
+    //     })
+    //     .catch((error) => console.log(error));
+    // };
```
Comment (elysemontano) [inline app/javascript/components/App.js]: Please remove commented out code.

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -10,8 +10,23 @@ import LeaderBoard from "./pages/LeaderBoard";
 import MetricsNew from "./pages/MetricsNew";
 import MetricsShow from "./pages/MetricsShow";
 import MetricsEdit from "./pages/MetricsEdit";
+// import MockMetrics from "./MockMetrics"
```
Comment (elysemontano) [inline app/javascript/components/App.js]: This import should not be commented out based on the Trello card.

**Review by elysemontano — CHANGES_REQUESTED** [review]
I left a few comments and I am seeing testing and modifications on other files that are not on the Trello card for this PR.  Please clean this up and I will approve.

File: app/javascript/components/App.js (hunk 1 above)
Comment (Mannyplg) [reply app/javascript/components/App.js]: Resolved.

File: app/javascript/components/App.js (hunk 2 above)
Comment (Mannyplg) [reply app/javascript/components/App.js]: Resolved.

**Review by elysemontano — APPROVED** [review]

**Pull request merged**